README.adoc

```diff
@@ -279,6 +279,11 @@ The base command is different whether you choose guice flavor or spring :
 
 DOMAIN: is the domain you want to add.
 
+Note: Using docker, one can add an environment variable holding the domain to be created. This
```

Comment:
that's not a hack, I just said that it already worked when used the right way.
That's great to add env, no problem with that.